elasticsearch/src/test/java/docs/javadsl/ElasticsearchV5Test.java

```diff
@@ -414,18 +414,16 @@ public void testUsingSearchParams() throws Exception {
     searchParams.put("query", "{\"match_all\": {}}");
     searchParams.put("_source", "[\"id\", \"a\", \"c\"]");
 
-    List<TestDoc> result =
+      // These documents will only have property id, a and c (not
```

Comment:
Is the positioning of this comment correct? Worth removing the `(not` part in the context of this example?

Reply:
Same for `*V7Test`